packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -15,12 +15,19 @@ import 'toggleable.dart';
 /// A material design checkbox
 ///
 /// The checkbox itself does not maintain any state. Instead, when the state of
-/// the checkbox changes, the component calls the `onChange` callback. Most
-/// components that use a checkbox will listen for the `onChange` callback and
-/// rebuild the checkbox with a new `value` to update the visual appearance of
+/// the checkbox changes, the widget calls the [onChanged] callback. Most
+/// widgets that use a checkbox will listen for the [onChanged] callback and
+/// rebuild the checkbox with a new [value] to update the visual appearance of
 /// the checkbox.
 ///
-/// <https://www.google.com/design/spec/components/lists-controls.html#lists-controls-types-of-list-controls>
+/// Requires one of its ancestors to be a [Material] widget.
+///
+/// See also:
+///  * [Radio]
```

Comment:
you need a newline before lists otherwise it ends up being one long paragraph (there are a number of cases of this being messed up that i wrote before discovering this)